src/docs/ftw-content/hosted-translations/index.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Hosted translations with Asset Delivery API
+slug: hosted-translations
+updated: 2022-05-13
+category: ftw-content
+ingress:
+  This article describes how hosted translations work in Flex Template
+  for Web (FTW).
+published: true
+---
+
+Starting from 2022-05, FTW template translations can be managed both in
+the built-in translation files and in Flex Console. This article
+describes how the FTW template uses the hosted translations and merges
+them with the built-in translations
+
+_**Note:** If you want to implement this feature into your pre-version
+XX.XX FTW template, you can see the necessary modifications in the PRs
+for [ftw-daily](https://github.com/sharetribe/ftw-daily/pull/1510). Read
+more:_
+
+- _[Translations in Flex Console](/concepts/translations/)_
+- _[How built-in translations work in the FTW templates](ftw/how-to-change-ftw-bundled-ui-translations/)_
+
+## Hosted translations
+
+**Hosted translations** are translations that the marketplace operator
+can edit in Flex Console. The client app then needs to fetch them using
+the Asset Delivery API.
+
+It is good to note that even if the operator adds some hosted
+translations using Flex Console, the FTW template still needs to have a
+built-in translation file for the translation keys that do not have a
+value in the hosted asset. That way, the UI can still render something
+meaningful for the parts of the page that the operator has not modified.
+
+There are two ways to fetch translation assets using Asset Delivery API:
+by version or by alias.
+
+#### Fetching translations by version
+
+All assets are identifiable by their version, and versions are
+immutable. Therefore if you fetch assets by version, they can be cached
+for an extended period of time. Read more about [caching assets]() TODO:
+LINK. When fetching translations by version, they are cached for an
+extended period of time, which helps to avoid unnecessary data loading.
```

Comment:
Maybe one more sencence?
"Since Asset Delivery API sets ***Cache-Control*** header for these responses, the browser knows to cache these responses on its own."